dbos/_sys_db.py

```diff
@@ -796,6 +828,34 @@ def record_operation_result(
                 raise DBOSWorkflowConflictIDError(result["workflow_uuid"])
             raise
 
+    def record_child_workflow(
+        self,
+        parentUUID: str,
+        childUUID: str,
+        functionID: int,
+        functionName: str,
+        conn: Optional[sa.Connection] = None,
```

Comment:
This argument isn't needed I think--this is never called as part of another transaction.